src/pages/index.tsx

```diff
@@ -1,23 +1,33 @@
 import type { NextPage } from 'next';
+import { GetServerSideProps } from 'next';
 
-// import { fetchApiRest } from '../requests/strapi-api';
-import { ArticleResponse } from '../interfaces/responses/article-response';
+import { Article } from '../interfaces/article';
 
 interface HomeProps {
-    allArticles: ArticleResponse[];
+    articles: Article[];
 }
 
-const Home: NextPage<HomeProps> = () => (
+const Home: NextPage<HomeProps> = ({ articles }) => (
     <>
         <h1>MongoDB Developer Center</h1>
+        <h2>Articles</h2>
+        <ul>
+            {articles.map(a => (
+                <li key={a.slug}>
+                    <h3>{a.name}</h3>
+                    <p>{a.description}</p>
+                </li>
+            ))}
+        </ul>
     </>
 );
 
 export default Home;
 
-// export async function getServerSideProps({}) {
-//     const allArticles = (await fetchApiRest('articles')) || [];
-//     return {
-//         props: { allArticles },
-//     };
-// }
+export const getServerSideProps: GetServerSideProps = async ({}) => {
+    const articlesModule = await import('../requests/articles'); // Dynamic import to allow unit testing without importing this.
```

Comment:
Not exactly sure what the implications of this are on performance. I did this to keep jest from trying to instantiate the apollo client (I was having a lot of trouble mocking the module...), which it tried to do when the import was at the top of the file. I'm hoping the module will be loaded at build time rather than runtime this way, but not sure if that's the case.

If someone knows how to better mock it out with jest, that would be great.

Reply:
I din't understand the question, matt.